test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -1320,6 +1320,70 @@ def test_text_conditional_1(self):
         circuit = QuantumCircuit.from_qasm_str(qasm_string)
         self.assertEqual(str(_text_circuit_drawer(circuit, vertical_compression='low')), expected)
 
+    def test_text_conditional_reverse_bits_true(self):
+        """ Conditional drawing with 1-bit-length regs."""
+        cr = ClassicalRegister(2, 'cr')
+        cr2 = ClassicalRegister(1, 'cr2')
+        qr = QuantumRegister(3, 'qr')
+        circuit = QuantumCircuit(qr, cr, cr2)
+        circuit.h(0)
+        circuit.h(1)
+        circuit.h(2)
+        circuit.x(0)
+        circuit.x(0)
+        circuit.x(2).c_if(cr, 2)
+        circuit.measure(2, 1)
```

Comment:
Can you move the conditional gate to after the measurement please? Is a more likely scenario

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5938/commits/ed8beeba922bd94a31ec366507bfc60e1a4c2202

Reply:
@enavarro51 looks like you've done it in the mpl test but not the text test?

Reply:
Ah, so it is. Done in https://github.com/Qiskit/qiskit-terra/pull/5938/commits/6e4587854490ecb6461cb592e515864c3fccb4b6

Reply:
Perfect, thank you!